src/types/annotator.js

```diff
@@ -26,7 +26,7 @@
 /**
  * An object representing an annotation in the document.
  *
- * @typedef AnnotationData
+ * @typedef AnnotationSafeData
```

Comment:
I do think it would be good to clarify the difference between this type and the API `Annotation`, and to use a different name. I'm not sure about the proposed name, it's a bit unwieldy.